manuscript.md

```diff
@@ -339,14 +345,40 @@ error---however this approach inevitably results in _overfitting_. One approach
 to avoid overfitting is using information criteria (*e.g.* AIC, BIC, MDL) based
 around the heuristic that good models maximize the ratio of information provided
 by the model to the number of parameters it has. However, when the intended
-use-case of a model is prediction the relevant form of validation is
-_predictive accuracy_, which should be tested with _crossvalidation_. Crossvalidation methods divide the
-original dataset into two---one which is used to fit the model (called the _training_ set) and one
- used to validate its predictive accuracy on the data that is hasn't "seen" yet
- (called the _test_ set) [@Bishop2006PatRec]. This procedure is often repeated
- for different subdivisions of the dataset [@Arlot2010SurCro].
-
-![TODO: validation figure caption](./figures/validation.png)
+use-case of a model is prediction the relevant form of validation is _predictive
+accuracy_, which should be tested with _crossvalidation_. Crossvalidation
+methods divide the original dataset into two---one which is used to fit the
+model (called the _training_ set) and one used to validate its predictive
+accuracy on the data that is hasn't "seen" yet (called the _test_ set)
+[@Bishop2006PatRec]. This procedure is often repeated for different subdivisions
+of the dataset [@Arlot2010SurCro].
+
+In the proof-of-concept, we used a neural-network to perform binary
+classification by predicting the presence/absence of an interaction between any
+two species. Many different metrics exist to validate the performance of a
+binary classifier. One approach is _accuracy_, the proportion of values it got
+correct.  However, consider what we know about interaction networks: they are
+often vary sparse, with connectance between $0.1$ and $0.3$. There are two ways
+for the model to be right: the model predicts an interaction and there is one,
+or the model predicts no interaction and there isn't one. If we built a model
+that always guesses there will be no interaction between two species, it will be
+correct in the majority of cases because the majority of potential interactions
+in a network typically do not exist. Therefore this "empty-matrix" model would
+always have an _accuracy_ of $1-C$, where $C$ is the observed connectance, which
+would almost always be greater than 50%! This emphasizes the importance of
+considering null models when validating a model's performance.
+
+One way to avoid is phenomena is to only consider the true-positive rate, which
+is the proportion of actually observed interaction that the model predicts
+correctly. A different metric is the true-skill statistic (TSS;
+@AssAccAllouche2006), which is related to ability to avoid both false-negative
+and false-positives. he performance of this proof-of-concept model in each of
+the metrics (accuracy, true positive, TSS) is shown in @fig:validation, and
+reflects that the proof-of-concept model works well with limited data, and would
```

Comment:
Can you explicitely mention the Albouy Gravel et al paper I linked to in #103 ? This would be a nice value added. When it's done, merge.